src/tests/Microsoft.PowerFx.Core.Tests/IntellisenseTests/SuggestTest.cs

```diff
@@ -34,13 +35,35 @@ private string[] SuggestStrings(string expression, PowerFxConfig config, string
             return intellisense.Suggestions.Select(suggestion => suggestion.DisplayText.Text).ToArray();
         }
 
-        private readonly PowerFxConfig _default = PowerFxConfig.BuildWithEnumStore(null, new EnumStoreBuilder().WithDefaultEnums());
+#pragma warning disable IDE0025 // Use Expression body for properties
+#pragma warning disable SA1300  // Element __ should begin with uppercase letter
```

Comment:
For non-public-surface items, rename instead of disabling the style check if possible.